docs/docs-content/clusters/data-center/nutanix/nutanix.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: "Nutanix"
+title: "Nutanix"
+description: "Learn how to configure Nutanix and create Nutanix clusters in Palette."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["data center", "nutanix"]
+---
+
+
+[Nutanix](https://www.nutanix.com/what-we-do) is a hyper-converged infrastructure platform that combines storage, compute, and networking into a single integrated system. Palette provides a generic framework built upon the open-source [Cluster API (CAPI)](https://cluster-api.sigs.k8s.io) initiative to support adding cloud providers. As a technical preview implementing this framework, you can deploy Kubernetes clusters on Nutanix using Palette. Integration with Nutanix is achieved through a Private Cloud Gateway (PCG), which establishes a secure connection with Nutanix.
+
+The section below describes the overall workflow to register Nutanix with Palette and deploy a cluster using Palette. 
+
+## Workflow
+
+The following summary steps outline the CAPI cloud workflow illustrated in the diagram below. A system administrator performs the first step at the system level to make the cloud provider available to all the tenants. CAPI users perform the remaining steps in Palette and their local environment.
+
+![Diagram showing the overall CAPI workflow for users to construct self-hosted cloud solutions tailored to their environment.](/clusters_data-center_nutanix_workflow.png)
+
+1. A system administrator registers Nutanix with Palette by invoking system-level APIs. This includes defining the logo, required cloud account information and key-value pairs to create the Nutanix cloud account. If you do not have a cloud logo, Palette provides a default logo. 
+
+2. Create a self-hosted PCG by preparing YAML configuration templates and installing Palette agents for the PCG in your Kubernetes cluster using the kubectl commands provided in the Palette UI.
```

Comment:
Possibly clarify the steps little bit more. Or point to the docs where self-hosted PCG is mentioned

Reply:
Added a link to the PCG doc.